examples/CivisML_parallel_training.ipynb

```diff
@@ -50,6 +52,7 @@
     "import pandas as pd\n",
     "import numpy as np\n",
     "import warnings\n",
+    "# xgboost uses a cross validate func deprecated by scikit-learn, but we don't depend on it\n",
```

Comment:
"a cross validate func deprecated by scikit-learn" --> "the deprecated `sklearn.cross_validate` module"?